### ISSUE TITLE
Doesn't work in babel v7.0.0-beta.3 due to a breaking change

### ISSUE THREAD

**Issue by zalishchuk** [styled-components/babel-plugin-styled-components#96]
In [babel v7.0.0-beta.3](https://github.com/babel/babel/releases/tag/v7.0.0-beta.3) `metadata` properties were removed [#6343](https://github.com/babel/babel/pull/6343).

[Code that needs to be changed](https://github.com/styled-components/babel-plugin-styled-components/blob/439c0ed33336c06e73ee03f34c1f00fb44fb8aab/src/utils/detectors.js#L4).
```
Module build failed: TypeError: Cannot read property 'imports' of undefined
    at importLocalName (/Users/zalishchuk/Desktop/moscow/node_modules/babel-plugin-styled-components/lib/utils/detectors.js:15:45)
    at isStyled (/Users/zalishchuk/Desktop/moscow/node_modules/babel-plugin-styled-components/lib/utils/detectors.js:36:61)
    at exports.default (/Users/zalishchuk/Desktop/moscow/node_modules/babel-plugin-styled-components/lib/visitors/minify.js:20:110)
    at PluginPass.TaggedTemplateExpression (/Users/zalishchuk/Desktop/moscow/node_modules/babel-plugin-styled-components/lib/index.js:19:30)
    at newFn (/Users/zalishchuk/Desktop/moscow/node_modules/babel-traverse/lib/visitors.js:225:21)
    at NodePath._call (/Users/zalishchuk/Desktop/moscow/node_modules/babel-traverse/lib/path/context.js:64:19)
    at NodePath.call (/Users/zalishchuk/Desktop/moscow/node_modules/babel-traverse/lib/path/context.js:38:17)
    at NodePath.visit (/Users/zalishchuk/Desktop/moscow/node_modules/babel-traverse/lib/path/context.js:95:12)
    at TraversalContext.visitQueue (/Users/zalishchuk/Desktop/moscow/node_modules/babel-traverse/lib/context.js:140:18)
    at TraversalContext.visitSingle (/Users/zalishchuk/Desktop/moscow/node_modules/babel-traverse/lib/context.js:99:19)
    at TraversalContext.visit (/Users/zalishchuk/Desktop/moscow/node_modules/babel-traverse/lib/context.js:181:19)
    at Function.traverse.node (/Users/zalishchuk/Desktop/moscow/node_modules/babel-traverse/lib/index.js:91:17)
    at NodePath.visit (/Users/zalishchuk/Desktop/moscow/node_modules/babel-traverse/lib/path/context.js:102:18)
    at TraversalContext.visitQueue (/Users/zalishchuk/Desktop/moscow/node_modules/babel-traverse/lib/context.js:140:18)
    at TraversalContext.visitMultiple (/Users/zalishchuk/Desktop/moscow/node_modules/babel-traverse/lib/context.js:94:17)
    at TraversalContext.visit (/Users/zalishchuk/Desktop/moscow/node_modules/babel-traverse/lib/context.js:179:19)
    at Function.traverse.node (/Users/zalishchuk/Desktop/moscow/node_modules/babel-traverse/lib/index.js:91:17)
    at NodePath.visit (/Users/zalishchuk/Desktop/moscow/node_modules/babel-traverse/lib/path/context.js:102:18)
    at TraversalContext.visitQueue (/Users/zalishchuk/Desktop/moscow/node_modules/babel-traverse/lib/context.js:140:18)
    at TraversalContext.visitMultiple (/Users/zalishchuk/Desktop/moscow/node_modules/babel-traverse/lib/context.js:94:17)
    at TraversalContext.visit (/Users/zalishchuk/Desktop/moscow/node_modules/babel-traverse/lib/context.js:179:19)
    at Function.traverse.node (/Users/zalishchuk/Desktop/moscow/node_modules/babel-traverse/lib/index.js:91:17)
```

**Comment by zalishchuk** [comment]
Any progress on this?

**Comment by codename-** [comment]
Any ETA on npm release of this fix?

**Comment by mxstbr** [comment]
On it, totally forgot! 🙈

**Comment by mxstbr** [comment]
Released in `v1.3.0`!
